lib/vsc/utils/fancylogger.py

```diff
@@ -326,7 +327,10 @@ def raiseException(self, message, exception=None, catch=False):
             exception = self.RAISE_EXCEPTION_CLASS
 
         self.RAISE_EXCEPTION_LOG_METHOD(fullmessage)
-        raise exception, message, tb
+
+        err = exception(message)
```

Comment:
why is this not `exception(message, tb)`?

Reply:
I based this change on https://github.com/hpcugent/vsc-install#old-raise-syntax, not sure if what you propose is compatible with both Python 2 & 3... @hajgato?

Reply:
@stdweird  I have followed the recommendations on this page: https://portingguide.readthedocs.io/en/latest/exceptions.html#the-new-raise-syntax I do believe that your proposed syntax would not work in `Python 3`.

Reply:
I discussed this with @hajgato, `exception(message, tb)` is most likely not compatible with Python 3.

On the other hand, fiddling with `err.__traceback__` does not work in Python 2.6 (and it probably explains why the tests fail in Travis for Python 2.6 while they pass for Python 2.7).

@stdweird Do we still care about Python 2.6 support? If not, I'll just drop the Python 2.6 config in Travis...

Reply:
https://legacy.python.org/dev/peps/pep-3109/

Reply:
@boegel yes, please drop 2.6